### ISSUE TITLE
add command to tye for converting docker-compose to tye

### ISSUE THREAD

**Issue by spboyer** [dotnet/tye#530]
fancy stuff, hard, but would be awesome.

Checkout [kompose](https://kompose.io/) as inspiration. -> https://github.com/kubernetes/kompose

**Comment by GeraudFabien** [comment]
Yes i need it too. Month ago i wanted to use Tye but since it was too early i used a docker-compose. Now i want to try because developing in container and i need to update tye.
But maybe with https://github.com/dotnet/tye/issues/190. It would be awesome if the docker-compose can do the job.
